sdk/storage/storage-blob/README.md

```diff
@@ -114,50 +115,46 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
-## SDK Architecture
-
-The Azure Storage SDK for JavaScript provides low-level and high-level APIs.
+## Examples
 
-- BlobServiceClient, ContainerClient and BlobClient objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
+### Create the Blob service Client
 
-- The high-level APIs provide convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).
-
-## Code Samples
+Use the constructor to create a instance of `BlobServiceClient`.
 
 ```javascript
-const {
-  BlobServiceClient,
-  newPipeline,
-  SharedKeyCredential,
-  AnonymousCredential,
-  TokenCredential
-} = require("@azure/storage-blob");
-
-async function main() {
   // Enter your storage account name and shared key
   const account = "account";
   const accountKey = "accountkey";
 
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
+  const blobServiceClient = new BlobServiceClient(
+    `https://${account}.blob.core.windows.net`,
+    sharedKeyCredential
+  );
+```
 
-  // Use TokenCredential with OAuth token
-  const tokenCredential = new TokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
+### Create a new container
 
-  // Use AnonymousCredential when url already includes a SAS signature
-  const anonymousCredential = new AnonymousCredential();
+Use `BlobServiceClient.createContainerClient()` to create a new container.
 
-  // Use sharedKeyCredential, tokenCredential or anonymousCredential to create a pipeline
-  const pipeline = newPipeline(sharedKeyCredential);
+```javascript
+  // Create a container
+  const containerName = `newcontainer${new Date().getTime()}`;
+  const containerClient = blobServiceClient.createContainerClient(containerName);
 
-  // List containers
-  const blobServiceClient = new BlobServiceClient(
-    // When using AnonymousCredential, following url should include a valid SAS or support public access
-    `https://${account}.blob.core.windows.net`,
-    pipeline
+  const createContainerResponse = await containerClient.create();
+  console.log(
+    `Create container ${containerName} successfully`,
+    createContainerResponse.requestId
   );
+```
 
+### List the containers
+
+Use `BlobServiceClient.listContainersSegment()` function to iterate the containers.
+
+```javascript
   let marker;
   do {
     const listContainersResponse = await blobServiceClient.listContainersSegment(
```

Comment:
Why not use for loop with async iterator?

Reply:
The latest async iterator PRs are not merged yet. Will update this.

Reply:
The implementation will change but the examples look the same. Updated.